astropy/utils/xml/writer.py

```diff
@@ -148,6 +153,20 @@ def start(self, tag, attrib={}, **extra):
         return len(self._tags)
 
     @contextlib.contextmanager
+    def xml_escaping(self, enabled=True, clean_kwargs=None):
+        current_xml_escape_cdata = self.xml_escape_cdata
+        if not enabled:
+            if HAS_BLEACH:
+                if clean_kwargs is None:
+                    clean_kwargs = {}
+                self.xml_escape_cdata = lambda x: bleach.clean(x, **clean_kwargs)
+            else:
+                raise ValueError('bleach package is required when HTML escaping is disabled.\n'
+                                 'Use "pip install bleach".')
```

Comment:
The way I'm seeing it, it's just an internal utility (doesn't even have a docstring) so it doesn't matter if it doesn't work if its dependencies aren't met.

It just seems strange to me to have a function that takes a flag that basic means "this should work" or "this shouldn't work".